certified-connectors/TrueDialog SMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1762 @@
+          "201": {
+            "description": "Stop callback created"
+          }
+        },
+        "summary": "Stop Message Received",
+        "description": "This operation triggers a flow whenever end user texts in STOP or synonym",
+        "operationId": "Stop_received",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "accountId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Use Root Account ID in most cases to subscribe for notifications.",
+            "x-ms-summary": "Account ID for subscription"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "CallbackType": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "CallbackType",
+                  "title": "CallbackType",
+                  "default": 6,
+                  "x-ms-visibility": "internal"
+                },
+                "URL": {
+                  "type": "string",
+                  "description": "URL",
+                  "title": "URL",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal"
+                },
+                "Active": {
+                  "type": "boolean",
+                  "description": "Active",
+                  "title": "",
+                  "enum": [
+                    true,
+                    false
+                  ],
+                  "default": true,
+                  "x-ms-visibility": "internal"
+                }
+              },
+              "required": [
+                "URL",
+                "CallbackType",
+                "Active"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/account/{accountId}/callback/-12": {
+      "x-ms-notification-content": {
+        "description": "Delivery Notice",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "CallbackTimestamp": {
+              "type": "string",
+              "description": "Date and time the callback has been sent. Format: yyyy-MM-ddTHH:mm:ss",
+              "title": "Callback Timestamp"
+            },
+            "CallbackToken": {
+              "type": "string",
+              "description": "CallbackToken"
```

Comment:
@TrueDialogDynamics All the description values of properties are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
Fixed descriptions in [this commit](https://github.com/microsoft/PowerPlatformConnectors/pull/1337/commits/11f2fa878ad92d61167f1226749ff251ca69060b)